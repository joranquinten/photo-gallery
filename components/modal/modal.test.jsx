import React from 'react';
import { shallow } from 'enzyme';

import Modal from './modal';

describe('Modal', () => {
  it('should render without crashing', () => {
    expect(shallow(<Modal closeModal={() => {}}>Child</Modal>)).toMatchSnapshot();
  });
});
