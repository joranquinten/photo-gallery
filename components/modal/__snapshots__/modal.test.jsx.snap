// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal should render without crashing 1`] = `
<div
  className="jsx-96374703 modal-container"
  title=""
>
  <div
    className="jsx-96374703 modal"
  >
    <button
      className="jsx-96374703"
      onClick={[Function]}
      type="button"
    >
      Ã—
    </button>
    Child
  </div>
  <JSXStyle
    css="body{overflow:hidden;}.modal-container{position:absolute;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-box-pack:center;-webkit-justify-content:center;-ms-flex-pack:center;justify-content:center;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;top:0;bottom:0;height:100%;width:100%;background:rgba(0,0,0,0.8);}.modal{position:relative;background:#fff;width:auto;height:auto;padding:0;margin:0;}.modal>button{font-weight:600;font-family:Arial,sans-serif;font-size:2em;border:none;position:absolute;right:0;top:0;z-index:2;background:transparent;cursor:pointer;color:#1e7d74;}button:hover,button:focus,button:active{color:#2ec4b6;}"
    styleId="96374703"
  />
</div>
`;
